cli/lsp/urls.rs

```diff
@@ -59,18 +57,16 @@ impl LspUrlMap {
   }
 
   fn put(&mut self, specifier: ModuleSpecifier, url: Url) {
-    let specifier = Arc::new(specifier);
```

Comment:
Doing this doubles up on the memory of a full allocation of both maps of URLs.  It isn't specifically important, but the `Arc` only allocated the thread safe pointer.